server/services/crmDisplayService.ts

```diff
@@ -57,17 +57,15 @@ export default class CrmDisplayService {
     }
   }
 
-  getSections<T extends CrmResponse>(crmType: CrmType, sectionId: string, crmResponse: T): Array<Section> {
+  getSections<T extends CrmResponse>(crmType: CrmType, sectionId: string, crmResponse: T): Section[] {
```

Comment:
Can we keep this as Array, to keep it consistent will usage throughout the codebase.
```suggestion
  getSections<T extends CrmResponse>(crmType: CrmType, sectionId: string, crmResponse: T): Array<Section> {
```